src/libsyntax/diagnostics/plugin.rs

```diff
@@ -82,10 +82,10 @@ pub fn expand_register_diagnostic<'cx>(ecx: &'cx mut ExtCtxt,
         token_tree.get(1),
         token_tree.get(2)
     ) {
-        (1, Some(&TokenTree::Token(_, token::Ident(ref code))), None, None) => {
+        (1, Some(&TokenTree::Token(_, token::Ident(ref code, false))), None, None) => {
```

Comment:
Looks like the rawness property is not relevant to diagnostics/plugin.rs, these `false`s should probably be `_`.

Reply:
Is this a huge deal either way? These are internal macros, aren't they?

Reply:
>Is this a huge deal either way?

No :)